internal/pkg/service/cli/cmd/dbt/generate/source/cmd.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 type Flags struct {
-	StorageAPIHost configmap.Value[string] `configKey:"storage-api-host" configShorthand:"H" configUsage:"storage API host, eg. \"connection.keboola.com\""`
-	TargetName     configmap.Value[string] `configKey:"target-name" configShorthand:"T" configUsage:"target name of the profile"`
+	StorageAPIHost  configmap.Value[string] `configKey:"storage-api-host" configShorthand:"H" configUsage:"storage API host, eg. \"connection.keboola.com\""`
+	StorageAPIToken configmap.Value[string] `configKey:"storage-api-token" configShorthand:"t" configUsage:"storage API token from your project"`
```

Comment:
I moved `StorageAPIToken` from GlobalFlags to commands structs where it are needed. We don't want `StorageAPIToken` and `StorageAPIHost` in the GlobalFlags struct.